sdk/confidentialledger/Azure.Security.ConfidentialLedger/tests/ConfidentialLedgerClientLiveTests.cs

```diff
@@ -212,6 +214,7 @@ public async Task CreateAndGetAndDeleteUser()
         }
 
         [RecordedTest]
+        [LiveOnly]
```

Comment:
Why is this live only?

Reply:
There is some weirdness with the Proxy config in this scenario, since this test doesn't need the custom transport config but all the others do. Instead of special casing it I just marked it LiveOnly for now.